Makefile

```diff
@@ -441,3 +451,19 @@ cpplint:
 lint: jslint cpplint
 
 .PHONY: lint cpplint jslint bench clean docopen docclean doc dist distclean check uninstall install install-includes install-bin all staticlib dynamiclib test test-all test-addons build-addons website-upload pkg blog blogclean tar binary release-only bench-http-simple bench-idle bench-all bench bench-misc bench-array bench-buffer bench-net bench-http bench-fs bench-tls
+
+## Build data tarballs
+data-tarballs: $(LEDATATGZ) $(BEDATATGZ)
+
+$(LEDATATGZ) $(BEDATATGZ): node
```

Comment:
Depending on how node has been built, `icupkg` may not have been built. Instead of depending on `node`, maybe we could make these two targets depend on `icupkg`? In the current state of the build system, an `icupkg` target  would depend on calling `./configure --with-intl=small-icu` first.